### PR TITLE
Updatedistancescript

### PR DESCRIPTION
**Pull request by MackJoly**
* **Please check if the PR fulfills these requirements**
- [X] The commit messages follows our guidelines (explicit, in english)
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [X] Bug fix: update all distance script
- [ ] New features:
- [ ] Documentation
- [ ] Optimization

* **Other information**:

### CODE REVIEW

**Review by AxelRinghausen — CHANGES_REQUESTED** [review]

File: scripts/read_steps.py [hunk 1]
```diff
@@ -33,4 +33,7 @@
     # model2 = model.copy()
     # assert model == model2
 
-    model._check_platform()
\ No newline at end of file
+#     model._check_platform()
+
+step = volmdlr.step.Step('C:\\Users\\Mack\\Downloads\\boite de vitesse DB45 HR18DD.stp')
```
[the remark this answers is not in the document]
Comment (MackJoly) [reply scripts/read_steps.py]: Thank you, i forgot it. It was a personal usecase

**Review by AxelRinghausen — DISMISSED** [review]

**Review by AxelRinghausen — APPROVED** [review]

**Comment by sbendjebla** [comment]
@MackJoly is this PR still useful ?

**Comment by MackJoly** [comment]
> @MackJoly is this PR still useful ?

Yes, it is an update of distance script used by drone

**Comment by sbendjebla** [comment]
@MackJoly  So, can you resolve conflicts, please ?

**Review by AxelRinghausen — APPROVED** [review]

**Comment by sbendjebla** [comment]
@MackJoly This PR needs to be merged or not ?

**Comment by MackJoly** [comment]
It could, it is an evolution of script distance that were incorrect, it needs to be checked i think

**Pull request closed without merging**